### ISSUE TITLE
Celo: <Issue Title>

### ISSUE THREAD

**Issue by CryptoZach** [celo-org/blockscout#925]
*Describe your issue here.*

### Environment
* Elixir Version: 1.13.1
* Erlang Version: 24
* BlockScout Version: fb1d16d3e7eb0e65d6f3e468e1530a0080e2f9e4

* User Agent: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36`

The voting rewards listed on the [Celo explorer](https://explorer.celo.org/mainnet/stats)(61k CELO) epoch rewards tab is different from the number shown on the epoch block explorer (https://explorer.celo.org/mainnet/block/0xdfb2982444336515c060f1a0f1b1f361c292b334a2216667427556865fff9399/epoch-transactions)(30.8k CELO). I’m pretty sure the Celo explorer stats page is wrong and mistakenly adds payments to the reserve booster or community fund as payments to voters, and also mistakenly labels validator cUSD payments as CELO payments. 

Can you please check on this and let me know what is causing the mismatch?

**Comment by 0xarthurxyz** [comment]
Thanks for sharing this here @CryptoZach! I'm  sorry for the delayed response.
@shazarre kindly agreed to take a look into this and get back to you.
Really appreciate you taking the time to log this here 💯

**Comment by 0xarthurxyz** [comment]
Hey @CryptoZach, thanks for your patience. I briefly wanted to get back to you on your query. Unfortunately @shazarre was OOO for a personal emergence recently, so I took a look at your question.

# Voter rewards APR

**TLDR**:

- Our APR number on the stats page is "wrong". We need to fix the calculation and appreciate you pointing that out 🙏 
- You should imply the APR as you suggested. For example, "epoch rewards" (27,711) divided by "locked and voting" CELO (268.3m) times a year (365 days): ~3.67% (=27,711 / 268,300,000 x 356 x 100)

**Context**:

- We are calculating an arithmetic mean (or "average") APR across all accounts that vote (or "stake") in [staking_rate.sql](https://github.com/celo-org/dbt-aggregate/blob/0a157c4a958142b8319631e1c9ebb6a8a6affbbb/models/staking_rate.sql#L55C1-L59C1)
  <img width="559" alt="image" src="https://github.com/celo-org/dbt-aggregate/assets/46296830/ac2891d0-5468-4454-a90f-6ca20e13c5a4">
- This is not very useful because it skews based on the distribution. For example, if you have:
  - 999,999 accounts that vote 1 CELO for a validator that isn't elected, they earn 0% APR
  - 1 whale account that votes 1,000,000 CELO for a validator that is elected, they earn the theoretical optimum per epoch (say 3.8%)
  - the arithmetic mean would be skewed and equal to 0.000...038% (= 999,999 x 0% + 1 x 3.8%) / 1,000,000 = 0.000000038%)
- A better calculation would be to use a weighted average

**Todos**:

- [ ] Convert APR calculation into a weighted average or simple "epoch rewards divided by locked and voting Celo". For example, "epoch rewards" (27,711) divided by "locked and voting" CELO (268.3m) times a year (365 days): ~3.67% (=27,711 / 268,300,000 x 356 x 100)
 - [ ] Reflect change in chart

# Voter rewards

**TLDR**: 
- The "voter rewards" in the "Distribution by Epoch" chart are incorrect:
  <img width="575" alt="image" src="https://github.com/celo-org/celo-labs/assets/46296830/e581d6a5-4bb7-4000-80ff-6b96eeb7f408">

**Context**:
- The calculation is incorrect in [distribution_by_epoch.sql](https://github.com/celo-org/dbt-aggregate/blob/0a157c4a958142b8319631e1c9ebb6a8a6affbbb/models/distribution_by_epoch.sql#L65C5-L65C39).
- Currently the calculation uses `rate` (which is the USD/CELO exchange rate) to convert the epoch reward into $. 
- Generally, the rewards should be expressed in CELO or might need to be multiple by `rate` rather than divided, if we indeed wanted rewards in $.

```diff
- vr.amount / rate as voter_rewards,
+ vr.amount as voter_rewards,
```

**Todos**:

- [ ] fix dbt query [distribution_by_epoch.sql](https://github.com/celo-org/dbt-aggregate/blob/0a157c4a958142b8319631e1c9ebb6a8a6affbbb/models/distribution_by_epoch.sql#L65C5-L65C39)
- [ ] fix chart
